CMSIS/NN/Include/arm_nnfunctions.h

```diff
@@ -62,15 +74,20 @@
    * Pre-processor Macros
    * ------------
    *
-   * Each library project have differant pre-processor macros.
+   * Each library project have different pre-processor macros.
    *
    * - ARM_MATH_DSP:
    *
    * Define macro ARM_MATH_DSP, If the silicon supports DSP instructions.
    *
+   * - ARM_MATH_MVEI:
+   *
+   * Define macro ARM_MATH_MVEI, If the silicon supports M-Profile Vector Extension.
+   *
    * - ARM_MATH_BIG_ENDIAN:
    *
-   * Define macro ARM_MATH_BIG_ENDIAN to build the library for big endian targets. By default library builds for little endian targets.
+   * Define macro ARM_MATH_BIG_ENDIAN to build the library for big endian targets. This is supported only for the legacy functions i.e, functions targetted at
+   * TensorFlow Lite do not support big endianness. By default library builds for little endian targets.
```

Comment:
I've not seen this limitation, where did u find this?

Reply:
@freddan80  it is not a limitation of TFL.. It is of the intrisics that we use which assumes the data to be little endian..